frontend/packages/console-app/src/components/tour/steps/StepFooter.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { Flex, FlexItem, Button } from '@patternfly/react-core';
+
+type StepFooterProps = {
+  primaryButton: {
+    name: string;
+    callback: () => void;
```

Comment:
Prefer callbacks to have prefix `on`. For example:
```suggestion
    onClick: () => void;
```